src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,16 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {
+      todos.map(todo => (
+        <TodoInfo todo={todo} />
+      ))
```

Comment:
`.map` inside JSX -> requires `key` attribute
```suggestion
      todos.map(todo => (
        <TodoInfo key={...} todo={todo} />
      ))
```